SingularityBase/src/main/java/com/hubspot/singularity/expiring/SingularityExpiringBounce.java

```diff
@@ -20,6 +20,34 @@ public String getDeployId() {
     return deployId;
   }
 
+  public static Optional<SingularityExpiringBounce> withDefaultExpiringMillis(
```

Comment:
we should make a `SingularityExpiringBounceBuilder` object instead... that way you can do something like:

```java
SingularityExpiringBounceBuilder.buildUpon(myExpiringBounceObject).setDurationMillis(Optional.of(7)).build()
```